storage/google/cloud/storage/client.py

```diff
@@ -60,6 +62,22 @@ def __init__(self, project=None, credentials=None, _http=None):
         self._connection = Connection(self)
         self._batch_stack = _LocalStack()
 
+    @classmethod
+    def create_anonymous_client(cls):
+        """Factory: return client w/ anonymous credentials.
```

Comment:
Please use `with` (throughout). Our (internal) documentation style guide discourages these kinds of abbreviations.

Reply:
93eec3e